src/components/forms/FormEditor.tsx

```diff
@@ -97,6 +100,24 @@ const FormEditor = ({ form }: FormEditorProps) => {
     );
   };
 
+  const onCreateFormTemplate = async () => {
+    const newForm: FormCreate = {
+      title: templateTitle, // her kommer det en popup der man kan skrive inn navnet til templaten
```

Comment:
```suggestion
      title: templateTitle,
```

Det er ikke nødvendig med kommentar her